server/routes/index.ts

```diff
@@ -298,6 +298,34 @@ export function defineRoutes(router: IRouter, esClient: IClusterClient) {
       }
     }
   );
+
+  router.put(
+    {
+      path: `${API_PREFIX}/configuration/audit/config`,
+      validate: {
+        body: schema.any(),
+      },
+    },
+    async (context, request, response) => {
+      const client = esClient.asScoped(request);
+      let esResp;
+      try {
+        esResp = await client.callAsCurrentUser('opendistro_security.audit', {
+          body: request.body,
+        });
+        return response.ok({
+          body: {
+            message: esResp.message,
+          },
+        });
+      } catch (error) {
+        return response.custom({
+          statusCode: error.statusCode,
+          body: error.message,
```

Comment:
Why not leverage `parseEsErrorResponse` to have it trying to parse error?

Reply:
Sure. I believe this is a new function introduced and was not aware of it during my PR.